satellite/orders/service.go

```diff
@@ -62,6 +62,32 @@ func (service *Service) saveSerial(ctx context.Context, serialNumber storj.Seria
 	return service.orders.CreateSerialInfo(ctx, serialNumber, bucketID, expiresAt)
 }
 
+func (service *Service) updateBandwidth(ctx context.Context, bucketID []byte, limits []*pb.AddressedOrderLimit) error {
+	if len(limits) == 0 {
+		return nil
+	}
+	var bucketAllocation int64
+	var action pb.PieceAction
+	nodesAllocation := make(map[storj.NodeID]int64)
+	for _, limit := range limits {
+		if limit != nil {
+			bucketAllocation += limit.Limit.Limit
+			nodesAllocation[limit.Limit.StorageNodeId] = limit.Limit.Limit
+			action = limits[0].Limit.Action
```

Comment:
There is a test panicking with nil reference here. My guess is that `limit[0]` is nil.

Reply:
Strange because earlier I have len(limits) == 0 but i will fix it because I made mistake during copy/paste.